requirements/edx/base.txt

```diff
@@ -90,10 +92,11 @@ django-memcached-hashring==0.1.2
 python-openid==2.2.5
 python-dateutil==2.1
 
-# This module gets monkey-patched in third_party_auth.py to fix a Django 1.8 incompatibility.
-# When this dependency gets upgraded, the monkey patch should be removed, if possible.
-# We can also remove the fix to auth_url in third_party_auth/saml.py when that fix is included upstream.
-python-social-auth==0.2.12
+# we have to install social-auth-app-django from fork to get over missing migration
+# issue https://github.com/python-social-auth/social-app-django/issues/28
+# once migrations are run we can switch back to base package(social-auth-app-django==1.0.1)
+-e git+https://github.com/ziafazal/social-app-django.git@8f4688076eab6d9b6fb71bb045e779b6c627835f#egg=social-auth-app-django
```

Comment:
I was trying different things however you have to look at final result. This Fix is for our current scenario which is upgrading from python-social-auth 2.12. which mean we assume we have these 3 migrations
default	0001_initial
default	0002_add_related_name
default	0003_alter_email_max_length